### DISCUSSION TITLE
Baked queries

### DISCUSSION THREAD

**Discussion by TomeCirun** [ckan/ckan#6918] [Ideas]
@wardi @amercader,  can we implement such things in ckan core,  if yes, where? I was wondering if this is going to improve ckan performance.

Thanks

Here is the [link](https://docs.sqlalchemy.org/en/14/orm/extensions/baked.html)
![image](https://user-images.githubusercontent.com/72216462/174323733-9c628c86-288f-4891-859f-5fd0c72558b5.png)

**Comment by wardi** [comment]
If we're making thousands of short selects then I think we're doing SQL wrong in the first place. If there are places in the code that could benefit from baked queries then it might be better to change the query pattern instead.

**Reply by wardi** [reply]
Interesting to learn about this feature though. We should experiment the next time we're working on a db performance issue.
